src/core/components/primitive-model.jsx

```diff
@@ -50,7 +50,7 @@ export default class Primitive extends Component {
       </span>
 
     return <span className="model">
-      <ModelCollapse title={titleEl} expanded={depth <= expandDepth} collapsedContent="[...]" hideSelfOnExpand={expandDepth !== depth}>
+      <ModelCollapse title={titleEl} expanded={depth <= expandDepth} collapsedContent="[...]" hideSelfOnExpand={false}>
```

Comment:
`hideSelfOnExpand` can be removed, as it's `false` by default in the `ModelCollapse` component:
```js
<ModelCollapse title={titleEl} expanded={depth <= expandDepth} collapsedContent="[...]"
```

Reply:
Hey @glowcloud 

I have resolved the mentioned change. Please have a look at the updated PR.